docs/src/examples/State-feedback Abstraction PWA System.jl

```diff
@@ -78,12 +76,12 @@ X_step = SVector(1.0 / n_step, 1.0 / n_step)
 nx = size(concrete_system.resetmaps[1].A, 1)
 P = (1 / nx) * diagm((X_step ./ 2) .^ (-2))
 state_grid = DO.GridEllipsoidalRectangular(X_origin, X_step, P, concrete_system.ext[:X]);
+opt_sdp = optimizer_with_attributes(SDPA.Optimizer, MOI.Silent() => true)
 
 optimizer = MOI.instantiate(AB.EllipsoidsAbstraction.Optimizer)
 MOI.set(optimizer, MOI.RawOptimizerAttribute("concrete_problem"), concrete_problem)
 MOI.set(optimizer, MOI.RawOptimizerAttribute("state_grid"), state_grid)
 MOI.set(optimizer, MOI.RawOptimizerAttribute("sdp_solver"), opt_sdp)
-MOI.set(optimizer, MOI.RawOptimizerAttribute("ip_solver"), opt_ip)
 
 # Build the state-feedback abstraction and solve the optimal control problem by through Dijkstra's algorithm [2, p.86].
 using Suppressor
```

Comment:
Maybe this could be removed, if the output of SDPA is now fixed.

Reply:
in fact this solver was used for the bounding box function which in fact do not need a solver.
That's why I decided to remove it

Reply:
or you are talking about the Suppressor package ?

Reply:
Yes, I'm talking about the `Suppressor` actually.
It was added because we couldn't make SDPA silent but it seems to be fixed